### PR TITLE
feat: add user keys

### PR DESCRIPTION
**Pull request by caarlos0**
This PR allows the user to add its own keys to their charm account.  

This includes yubikeys or anything else the x/crypto libs support.  

This is still not using the actual user keys for anything... for that we would probably want to support ssh agents et al as well

Likely depends on #121

### CODE REVIEW

**Comment by bashbunni** [comment]
yubikeys >:o

File: client/client.go [hunk 1]
```diff
@@ -92,23 +98,56 @@ func NewClient(cfg *Config) (*Client, error) {
...
+		if err != nil {
+			return nil, err
+		}
+		pkam = append(pkam, m)
+	}
+
+	if len(pkam) == 0 {
+		return nil, charm.ErrMissingSSHAuth
+	}
+	log.Println("KEYS", len(pkam))
```
Comment (aymanbagabas) [inline client/client.go:121]: extra print statement

**Review by aymanbagabas — APPROVED** [review]

**Comment by caarlos0** [comment]
@toby would love your feedback in this one as  well

**Comment by caarlos0** [comment]
changed it to a draft because a lot of things don't work, and some can't really work the way we're doing things today

so... back to the drawing board :D

**Pull request closed without merging**